common/src/java/com/zimbra/common/account/ZAttrProvisioning.java

```diff
@@ -6494,6 +6494,15 @@ public static TwoFactorAuthSecretEncoding fromString(String s) throws ServiceExc
     @ZAttr(id=1373)
     public static final String A_zimbraFeatureExternalFeedbackEnabled = "zimbraFeatureExternalFeedbackEnabled";
 
+    /**
+     * Enable/Disable blocked file types set in
+     * zimbraFileUploadBlockedFileTypes for uploading
+     *
+     * @since ZCS 9.0.0
+     */
+    @ZAttr(id=3092)
+    public static final String A_zimbraFeatureFileTypeUploadRestrictionsEnabled = "zimbraFeatureFileTypeUploadRestrictionsEnabled";
```

Comment:
are we planning to release this change in kepler patches? ldap changes are not possible to be released with patch system

Reply:
if this feature is targeted for patch release, please use LC and not ldap attribute.

Reply:
@Prashantsurana Can you please suggest.

Reply:
We can go with LDAP attributes.